subiquity/ui/views/ubuntu_pro.py

```diff
@@ -221,8 +221,8 @@ class UpgradeYesNoFormNoNetwork(UpgradeYesNoForm):
         _("Skip Ubuntu Pro setup for now"),
         help="\n"
         + _(
-            "Once you are connected to the Internet, you can"
-            " enable Ubuntu Po via the 'pro attach' command."
+            "Once you are connected to the internet, you can"
+            " enable Ubuntu Pro using the 'pro attach' command."
```

Comment:
Could you reflect these changes in the class docstring as well?

Reply:
Of course. Comment blindness :)